common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -151,6 +152,52 @@ public MsalOAuth2TokenCache(final Context context,
         );
     }
 
+    /**
+     * @param accountRecord : AccountRecord associated with the input credentials.
+     * @param credentials   : list of Credential which can include AccessTokenRecord, IdTokenRecord and RefreshTokenRecord.
+     *                      Note : Both IdTokenRecord and RefreshTokenRecord need to be non null. AccessTokenRecord can be optional.
+     * @throws ClientException : If the supplied Account or Credential are null or schema invalid.
+     */
+    public void saveCredentials(@NonNull final AccountRecord accountRecord,
```

Comment:
I'm thinking if we should override this class and add this method only in the inherited classs (i.e. MsalCppOAuth2TokenCache).

This way, we would be able to minimize exposing this to MSAL devs.

We can mark the original saveCredentials as protected.

Reply:
+1 Like this idea

Reply:
Thanks for the suggestion Dome. Addressed